build/version.go

```diff
@@ -37,7 +37,7 @@ func BuildTypeString() string {
 }
 
 // BuildVersion is the local build version
-const BuildVersion = "1.25.3-dev"
+const BuildVersion = "1.26.0-rc2"
```

Comment:
I don't think we want this, should be 1.27.0-dev, I think.

Reply:
Updated in https://github.com/filecoin-project/lotus/pull/11699/commits/a613651b3bf49e456d9e4a7381157b7409e9022c